model/mobile_trigger.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type MobileTrigger struct {
```

Comment:
Maybe a general `Text` and `Icon` field make sense. For post menu actions and all dropdown that seams useful.

Reply:
Yes, it is useful for all dropdowns. Once the functionality is more mature and I get to deal with the RPC problems, we can define properly each location.